.gitignore

```diff
@@ -5,6 +5,8 @@ __pycache__
 .env
 staticfiles/
 mysite.log
+settings.py
```

Comment:
Please don't do this. 
(Thank GOD I'm not Linux Torvalds)

Reply:
My bad :33, I did it for my convenience, forgot to change back. 
Just fixed it.